content/en/blog/_posts/2023-08-15-retroactive-default-storage-class-ga.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: blog
+title: "Kubernetes v1.28: Retroactive Default StorageClass move to GA"
+date: 2023-08-15
+slug: retroactive-default-storage-class-ga
+---
+
+**Author:** Roman Bednář (Red Hat)
+
+Announcing graduation to General Availability (GA) - Retroactive Default StorageClass Assignment in Kubernetes v1.28!
+
+We are thrilled to announce that the "Retroactive Default StorageClass Assignment" feature,
+introduced as an alpha in Kubernetes v1.25, has now graduated to GA and is officially part of the Kubernetes v1.28 release.
+This enhancement brings a significant improvement to how default StorageClasses are assigned to PersistentVolumeClaims (PVCs).
+
+With this feature enabled, you no longer need to create a default StorageClass first and then a PVC to assign the class.
+Instead, any PVCs without a StorageClass assigned will now be retroactively updated to include the default StorageClass.
+This enhancement ensures that PVCs no longer get stuck in an unbound state, and storage provisioning works seamlessly,
+even when a default StorageClass is not defined at the time of PVC creation.
+
+## What changed?
+
+The PersistentVolume (PV) controller has been modified to automatically assign a default StorageClass to any unbound
+PersistentVolumeClaim with the storageClassName set to null. Additionally, the PersistentVolumeClaim admission within
```

Comment:
```suggestion
PersistentVolumeClaim with the `storageClassName` not set. Additionally, the PersistentVolumeClaim
admission validation mechanism within
```
In the API response, I think the field is omitted when not set. Have I got that right?

Reply:
Correct, and the previous post which we reference here describes the difference between "not set" and `""` in more detail. Saying "not set" here is a more accurate summary of the new behavior for sure.